### PR TITLE
New python related updates

### PR DESCRIPTION
**Pull request by olichtne**
### Description
We're looking to use newer python internally and with that we need to do some updates.

### Tests
(Please provide a list of tests that prove that the pull
request doesn't break the stable state of the master branch. This should
include test runs with valid results for all of critical workflows.)

### Reviews
(Please add a list of reviewers that should check the validity and sanity of
this merge request before it's accepted. Use the `@username` syntax. If you
don't know who to mention just link `@all`.)

Closes: #

### CODE REVIEW

**Comment by olichtne** [comment]
tested in `J:9347833` and the warnings are gone.

**Comment by olichtne** [comment]
Most up to date tests are in: `J:9515688` and `J:9542445`

File: lnst/Recipes/ENRT/Ip6GreNetnsTunnelRecipe.py [hunk 1]
```diff
@@ -29,7 +29,7 @@
 class Ip6GreNetnsTunnelRecipe(
     MTUHWConfigMixin, PauseFramesHWConfigMixin, OffloadSubConfigMixin, BaseTunnelRecipe
 ):
-    """
+    r"""
```
Comment (jtluka) [inline lnst/Recipes/ENRT/Ip6GreNetnsTunnelRecipe.py:32]: This one is probably not required?

File: lnst/Recipes/ENRT/LinuxBridgeOverBondRecipe.py [hunk 2]
```diff
@@ -23,7 +23,7 @@
 
 
 class LinuxBridgeOverBondRecipe(CommonHWSubConfigMixin, OffloadSubConfigMixin, BaremetalEnrtRecipe):
-    """
+    r"""
```
Comment (jtluka) [inline lnst/Recipes/ENRT/LinuxBridgeOverBondRecipe.py:26]: This one is probably not required?

**Review by jtluka — APPROVED** [review]
Looks ok.

**Pull request merged**